hook.php

```diff
@@ -71,9 +71,13 @@ function plugin_escalade_install() {
          `reassign_tech_from_cat`                  INT(11) NOT NULL,
          `cloneandlink_ticket`                     INT(11) NOT NULL,
          `close_linkedtickets`                     INT(11) NOT NULL,
-         `use_assign_user_group`                   INT(11) NOT NULL,
```

Comment:
I do not understand why those ones have been removed... There are stillr eferences to them in the code, isn't it?